app/views/apps/show.html.haml

```diff
@@ -63,7 +63,7 @@
         %td= mail_to(engagement.contact.email, engagement.contact.name) if engagement.contact
         %td= [engagement.coach.try(:name),engagement.coaching_org.try(:name)].join(', ')
         %td= engagement.team_number
-        %td= engagement.student_names
+        %td <div class="members"><div class="member1">#{engagement.users[0].try(:name)} </div><div class="member2">#{engagement.users[1].try(:name)}</div><div class="member3">#{engagement.users[2].try(:name)}</div><div class="member4">#{engagement.users[3].try(:name)}</div><div class="member5">#{engagement.users[4].try(:name)}</div><div class="member6">#{engagement.users[5].try(:name)}</div></div>
```

Comment:
can we do some sort of loop here instead?

Reply:
The loop implementation is definitely the way to go but Julie and I have currently some issues displaying the names under the correct column "Developers". When we use the loop, names are displayed on top of the table, namely before "Selected". We will fix this during the week after consulting with Andrew.